packages/perseus/src/widgets/__shared__/noop-validator.tsx

```diff
@@ -8,11 +8,11 @@ import type {PerseusScore} from "../../types";
  * validation logic and the thing validating an exercise
  * should just know to skip these
  */
-function noopValidator(): PerseusScore {
+function noopValidator(points: number = 0): PerseusScore {
```

Comment:
The comment of this validator now seems slightly wrong. With the addition of a parameter we can make a validator that always returns "correct" (points).